frontend/src/pages/Visualizer/index.tsx

```diff
@@ -145,6 +145,7 @@ export default class Visualizer extends React.Component<VisualizerProps, Visuali
     }, {})
     // TODO add connection update info 
     const submittedStructure: ModelStructure = { 'layer': updatedLayers, 'connection': {} }
+    console.log(submittedStructure)
```

Comment:
Can remove such print function for debugging perpose.

Reply:
Of course we should